packages/gatsby-theme-newrelic/src/components/AnnouncementBanner.js

```diff
@@ -22,14 +22,19 @@ const findCurrentAnnouncement = (announcements) => {
   );
 };
 
-const createContentHash = (announcement) =>
-  btoa(
+const createContentHash = (announcement) => {
+  // Since bota is not available at build time, this will provide a
+  // fallback so that this function simply joins the items.
+  const btoa = global.btoa || ((arr) => arr);
```

Comment:
what is the `((arr) => arr)` doing?

Reply:
This is a fallback function that simply returns whatever is passed to it. If `createContentHash` is called and `btoa` doesn't exist, we would simply return whatever was passed to the function.

In this case, we're joining the `id`, `startDate`, and `endDate` with a `:` in between. If `btoa` exists, we'll create a hash of that string. If it doesn't, we'll just return the joined string without hashing it.

Reply:
okay! awesome, thank you